src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -231,7 +231,7 @@ private function extractFromAccessor($class, $property)
      */
     private function extractFromReflectionType(\ReflectionType $reflectionType)
     {
-        $phpTypeOrClass = (string) $reflectionType;
+        $phpTypeOrClass = method_exists($reflectionType, 'getName') ? $reflectionType->getName() : (string) $reflectionType;
```

Comment:
`\ReflectionType` does't have a `getName()` method according to the docs: http://php.net/manual/en/class.reflectiontype.php. Will this be added in the future perhaps?